### PR TITLE
D8CORE-4396: changes to remove focus for accessibility reason

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Removing the alt from images in a new list
- Removing the image from the tab order if using keyboard navigation

# Review By (Date)
- 7/8

# Criticality
- Med

# Urgency
-  Med

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Navigate to news page and then a news list in a single column. (try all the verify lines in both lists)
4. Verify the images don't have alt text.
5. Verify you can tab through and skip focus on the image.
6. Verify you can still click the link with the mouse.
7. Listen on voiceover to hear there is no image alt text read.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? - No

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [x] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [x] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- News and Default Lists of News

# Associated Issues and/or People
- D8CORE-4396

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: templates/components/news-list/news-item-list.twig [hunk 1]
```diff
@@ -68,7 +68,7 @@
     {# Add the image and link only if they both exist. #}
     {% if news_list_image|render|striptags('<drupal-render-placeholder><img><picture>')|trim is not empty -%}
       {%- if news_url is not empty -%}
-        <a {{ news_url_attributes }} href="{{ news_url }}" aria-hidden="true">
+        <a {{ news_url_attributes }} href="{{ news_url }}" aria-hidden="true" tabindex="-1">
```
Comment (pookmish) [inline templates/components/news-list/news-item-list.twig:71]: Didn't Kevin say this needed an `aria-hidden="true"` as well?

Comment (jenbreese) [reply templates/components/news-list/news-item-list.twig:71]: I see it on line 71. But I am on the phone…

Comment (jenbreese) [reply templates/components/news-list/news-item-list.twig:71]: On line 71, I see the link already had the `aria-hidden="true"`. I didn't have to add it.

Comment (pookmish) [reply templates/components/news-list/news-item-list.twig:71]: wow. i'm blind. Don't mind me.

**Review by pookmish — APPROVED** [review]

**Pull request merged**